### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by omar-dulaimi**
## Describe this PR

added:

- class validator generator
- zod generator
- trpc generator

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx
Comment (Jolg42) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`prisma-class-validator-generator`](https://github.com/omar-dulaimi/prisma-class-validator-generator): Emit TypeScript models from your Prisma schema with class validator validations ready.
```

**Review by Jolg42 — APPROVED** [review]

**Review by janpio — APPROVED** [review]

**Comment by omar-dulaimi** [comment]
Hello,

Should I submit other generators I built?

Or should I wait until this one gets merged?

**Comment by Jolg42** [comment]
@omar-dulaimi I think you can update this PR here 👍🏼

**Comment by omar-dulaimi** [comment]
@Jolg42 I have updated this PR with two more generators.

**Pull request merged**
